src/commands/emergency_solution.rs

```diff
@@ -45,5 +47,51 @@ where
 		+ 'static,
 	T::Solution: Send,
 {
-	todo!("not possible to implement yet");
+	let round = api
+		.storage()
+		.at(config.at)
+		.await?
+		.fetch_or_default(&runtime::storage().election_provider_multi_phase().round())
+		.await?;
+
+	let miner_solution = epm::fetch_snapshot_and_mine_solution::<T>(
+		&api,
+		config.at,
+		config.solver,
+		round,
+		config.force_winner_count,
+	)
+	.await?;
+
+	let ready_solution = miner_solution.feasibility_check()?;
```

Comment:
Because `MaxWinners` is read from the metadata and if one enables `--force-winner-count <val>` this might differ from the metadata and feasiblity_check reads the value from the metadata.

so either just do the feasibility_check if `force_winner_count.is_none()` or add the following in the beginning of this function:

```rust
	if Some(max_winners) = config.force_winner_count {
		static_types::MaxWinners::set(config.force_winner_count);
	}
```